ui/values/dimensions.go

```diff
@@ -65,10 +65,12 @@ var (
 	MarginPadding75       = unit.Dp(75)
 	MarginPadding80       = unit.Dp(80)
 	MarginPadding96       = unit.Dp(96)
+	DP85                  = unit.Dp(85)
```

Comment:
```suggestion
	MarginPadding85                  = unit.Dp(85)
```

right now it was confusing for me when i was trying to make some changes in my PR, a single refactor should convert everything at a later time, for now let's keep using margin, padding

Reply:
okay